stubs/docker/docker/models/containers.pyi

```diff
@@ -57,10 +57,16 @@ class Container(Model):
 class ContainerCollection(Collection[Container]):
     model: type[Container]
     def run(
-        self, image, command: Incomplete | None = None, stdout: bool = True, stderr: bool = False, remove: bool = False, **kwargs
+        self,
+        image,
```

Comment:
```suggestion
        image: str,
```
This is what the docstring says, any reason not to use it?

Reply:
Thank you, I have committed this change. I have a feeling that I tried this and it errored, but now @kasium 's work elsewhere has made this not error.